server/routes/viewingLicences/handlers/viewLicence.ts

```diff
@@ -26,9 +51,28 @@ export default class ViewAndPrintLicenceRoutes {
         )
       }
 
-      res.render('pages/view/view', { additionalConditions: groupingBy(licence.additionalLicenceConditions, 'code') })
+      res.render('pages/view/view', {
+        additionalConditions: groupingBy(licence.additionalLicenceConditions, 'code'),
+        warningMessage,
+      })
     } else {
       res.redirect(`/licence/view/cases`)
     }
   }
+
+  getFormattedLicenceDate(licence: Licence): string {
+    let licenceDate
+    switch (licence.statusCode) {
+      case LicenceStatus.IN_PROGRESS:
+        licenceDate = licence.dateCreated
+        break
+      case LicenceStatus.APPROVED:
+        licenceDate = licence.approvedDate
+        break
+      default:
+        licenceDate = licence.dateLastUpdated
```

Comment:
discussed with Tom, I'm going to add a new column, will probably be useful for other things as well

Reply:
added submitted date to licence and now use that if available. If not available as licence was submitted before column was added then don't display that part of message. Tom said it would be better to not display that display incorrect info.